main/src/main/java/mockit/internal/reflection/MethodReflection.java

```diff
@@ -7,11 +7,7 @@
 import static java.lang.reflect.Modifier.isPrivate;
 import static java.lang.reflect.Modifier.isStatic;
 
-import static mockit.internal.reflection.ParameterReflection.acceptsArgumentTypes;
-import static mockit.internal.reflection.ParameterReflection.getParameterTypesDescription;
-import static mockit.internal.reflection.ParameterReflection.hasMoreSpecificTypes;
-import static mockit.internal.reflection.ParameterReflection.indexOfFirstRealParameter;
-import static mockit.internal.reflection.ParameterReflection.matchesParameterTypes;
+import static mockit.internal.reflection.ParameterReflection.*;
```

Comment:
@yukkes Please don't use star imports.  Usually your IDE causes that, make sure intellij (going on a limb that is what you are using) is setup appropriately to stop doing that.  Its meant for use if something like 99 imports which isn't the case.